dashboard/src/pages/TreeDetails/Tabs/Tests/TestsTable.tsx

```diff
@@ -172,79 +280,87 @@ const TestsTable = ({ testHistory }: ITestsTable): JSX.Element => {
     [filterCount, intl, tableFilter.testsTable],
   );
 
-  const tableInfoElement = (
-    <div className="flex flex-col items-end">
-      <TableInfo
-        itemName="global.tests"
-        startIndex={startIndex + 1}
-        endIndex={endIndex}
-        totalTrees={data_len}
-        itemsPerPageValues={ItemsPerPageValues}
-        itemsPerPageSelected={itemsPerPage}
-        onChangeItemsPerPage={setItemsPerPage}
-        onClickBack={onClickGoBack}
-        onClickForward={onClickGoForward}
-      />
-    </div>
-  );
+  const table = useReactTable({
+    data,
+    columns,
+    onSortingChange: setSorting,
+    getCoreRowModel: getCoreRowModel(),
+    getPaginationRowModel: getPaginationRowModel(),
+    onPaginationChange: setPagination,
+    getSortedRowModel: getSortedRowModel(),
+    getFilteredRowModel: getFilteredRowModel(),
+    getRowCanExpand: _ => true,
+    getExpandedRowModel: getExpandedRowModel(),
+    onExpandedChange: setExpanded,
+    state: {
+      sorting,
+      pagination,
+      expanded,
+    },
+  });
 
-  const filteredData = useMemo(() => {
-    switch (tableFilter.testsTable) {
-      case 'all':
-        return data;
-      case 'success':
-        return data
-          ?.filter(tests => tests.pass_tests > 0)
-          .map(test => ({
-            ...test,
-            individual_tests: test.individual_tests.filter(
-              t => t.status?.toUpperCase() === StatusTable.PASS,
-            ),
-          }));
-      case 'failed':
-        return data
-          ?.filter(tests => tests.fail_tests > 0)
-          .map(test => ({
-            ...test,
-            individual_tests: test.individual_tests.filter(t => {
-              const result = t.status?.toUpperCase() === StatusTable.FAIL;
+  const groupHeaders = table.getHeaderGroups()[0].headers;
+  const tableHeaders = useMemo((): JSX.Element[] => {
+    return groupHeaders.map(header => {
+      return (
+        <TableHead key={header.id} className="border-b px-2 font-bold">
+          {header.isPlaceholder
+            ? null
+            : flexRender(header.column.columnDef.header, header.getContext())}
+        </TableHead>
+      );
+    });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
why are you disabling the eslint here? I never saw a place where this was a good solution? In any case, if you can't find a soution you can just memo the list item instead of the entire list